native/src/components/Header.tsx

```diff
@@ -184,14 +191,20 @@ const Header = ({
     ),
     renderItem(HeaderButtonTitle.Language, 'language', showItems, goToLanguageChange),
   ]
-
   const overflowItems = showOverflowItems
     ? [
         ...(shareUrl ? [renderOverflowItem(HeaderButtonTitle.Share, onShare)] : []),
         ...(!buildConfig().featureFlags.fixedCity
           ? [renderOverflowItem(HeaderButtonTitle.Location, () => navigation.navigate(LANDING_ROUTE))]
           : []),
         renderOverflowItem(HeaderButtonTitle.Settings, () => navigation.navigate(SETTINGS_ROUTE)),
+        ...[
+          content &&
+          (buildConfig().appName === 'IntegreatTestCms' || buildConfig().appName === 'Integreat') &&
```

Comment:
please use a buildConfig flag (`ttsEnabled` or sth similar) for tts instead of checking the app name here